internal/commands/reminder.go

```diff
@@ -215,27 +203,27 @@ func setRecurrence(incident model.Incident) time.Duration {
 }
 
 func sendNotification(ctx context.Context, app *app.App, incident model.Incident) {
+	logWriter := app.Logger.With(
+		log.NewValue("channelID", incident.ChannelId),
+		log.NewValue("channelName", incident.ChannelName),
+		log.NewValue("incident.Status", incident.Status),
+	)
+
 	err := postMessage(app, incident.ChannelId, statusNotify(incident))
 
 	if err != nil {
 		app.Logger.Error(
 			ctx,
 			log.Trace(),
 			log.Action("postMessage"),
-			log.NewValue("channelID", incident.ChannelId),
-			log.NewValue("channelName", incident.ChannelName),
-			log.NewValue("incident.Status", incident.Status),
 			log.NewValue("error", err),
 		)
 		return
 	}
 
-	app.Logger.Info(
+	logWriter.Info(
```

Comment:
Why Info instead of Debug here?

Reply:
Also, suggestion: move this to before the postMessage call, right after the declaration of logWriter.